projects/RPi/devices/RPi2/patches/bluez/bluez.patch

```diff
@@ -0,0 +1,58 @@
+The 32-bit SIOCGSTAMP has been deprecated. Use the deprecated name
```

Comment:
I guess this patch can be moved to `/packages/network/bluez/patches` and then should be removed from projects `/patches/bluez`.

Reply:
yes it can